sconfig_test.go

```diff
@@ -520,3 +554,13 @@ func executeCommand(root *cobra.Command, args ...string) error {
 
 	return err
 }
+
+func executeCommandWithOutput(root *cobra.Command, args ...string) (string, error) {
```

Comment:
I actually think it'd be better to just update `executeCommand` to return the output, and then do this when it's not needed. Do you mind making that update?

```go
_, err := executeCommand(...)

Reply:
💯